src/PortingAssistant.Client.NuGet/PortingAssistantNuGetHandler.cs

```diff
@@ -103,7 +103,7 @@ private async void Process(List<PackageVersionPair> packageVersions, string path
                 if (packageVersion != null && _compatibilityTaskCompletionSources.TryGetValue(packageVersion, out var packageVersionPairResult))
                 {
                     _logger.LogError($"Cound not find package {packageVersion} in all sources");
```

Comment:
We can comment out line 105 since the same message is added to exception that is being thrown

Reply:
Agree.  that makes more sense.  here is the new log.  

2022-02-16T22:17:59.8111936-08:00	FAIL	[PortingAssistant.Client.Analysis.PortingAssistantAnalysisHandler]	[0]	parse package version [package name - version] with error PortingAssistant.Client.Model.PortingAssistantClientException: Cannot find package [package name - version] -NUGET

![image](https://user-images.githubusercontent.com/98550439/154421162-ec4fbb16-178c-4ea4-a2bb-9d704bfa33d9.png)